lib/flex.js

```diff
@@ -52,7 +52,7 @@ const formatErrorSummary = (errors) => {
 
 const processWorkflow = async (context, registry, config) => {
   // go through the settings and register all the validators
-  await config.registerValidatorsAndActions(registry)
+  config.registerValidatorsAndActions(registry)
```

Comment:
I removed the `await` keyword here because the function `registerValidatorsAndActions` is not even async. It is redundant and confusing IMO.